playbooks/openshift/roles/bfs/tasks/build_origin_repo.yml

```diff
@@ -0,0 +1,50 @@
+- debug:
```

Comment:
Again troys instructions for origin this was missing from producing versioned rpms